src/Gamecure.Core/Configuration/ValidateApplicationConfig.cs

```diff
@@ -34,9 +34,10 @@ public async Task<AppConfigContext> OnInvoke(AppConfigContext context, ContextDe
         {
             return context with { Failed = true, Reason = $"{nameof(AppConfig.Container)} has not been set." };
         }
-        if (string.IsNullOrWhiteSpace(context.Configuration.Jira.Url))
+
+        if (context.Configuration.Jira.Enabled && string.IsNullOrWhiteSpace(context.Configuration.Jira.Url))
```

Comment:
check for null and then validate the url
`if (context.Configuration.Jira is not null && string.IsNullOrWhiteSpace(context.Configuration.Jira.Url))`